cpp/src/slice2cpp/CPlusPlusUtil.cpp

```diff
@@ -975,32 +975,6 @@ Slice::writeIceTuple(::IceInternal::Output& out, DataMemberList dataMembers, Typ
     out << ");" << eb;
 }
 
-bool
```

Comment:
These 2 `findMetadata` functions were unused.

There's another `findMetadata` you can see underneath these 2 that I'd like to remove in the future,
but it's used in lots of places. So I'll deal with it in a future PR.